tox.ini

```diff
@@ -31,29 +31,21 @@ passenv =
 description = Apply coding style standards to code
 deps =
     black
-    isort
+    ruff
```

Comment:
With ruff we still need black?
For example: https://beta.ruff.rs/docs/settings/#docstring-quotes

Reply:
Actually we most likely don't; I didn't think about it because I was keeping the changes as minimal as possible, but the `fmt` has been run by **ruff** only in all the PRs already; **black** seems to be redundant indeed!